Atomic/util.py

```diff
@@ -1096,3 +1099,18 @@ def kpod(cmd, storage=None, debug=None):
     if debug:
         write_out(" ".join(cmd))
     return check_output(_kpod, env=os.environ)
+
+
+def remove_skopeo_prefixes(image):
+    """
+    Remove prefixes used by skopeo but not expected in other image uses.
```

Comment:
The prefix are not really used by Skopeo itself, but we map them to the Skopeo options.  I'd just s|by|with|

I am still not sure this should be in util.py as it is specific to system containers, could we move it to syscontainers.py?  On the other hand, I don't want to hold this PR any longer, so we can address this later

Reply:
I'll update the docstring.